src/main/groovy/io/seqera/wave/service/persistence/WaveContainerRecord.groovy

```diff
@@ -70,7 +70,7 @@ class WaveContainerRecord {
     /**
      * The conda file associated with the request
      */
-    final String condaFile
+    String condaFile
```

Comment:
Looking better, don't love this. What if it's assigned truncating it ahead ?

Reply:
I thought so too, but truncation is only related to surrealdb
truncating it here will also impact any other database

Reply:
it depends. It may applied generally because it can be good to have protection to prevent dumping a large amount of data in the DB storage, irrespective of the engine

Reply:
but still 14kb is small